daprdocs/content/en/reference/components-reference/supported-state-stores/setup-jetstream-kv.md

```diff
@@ -0,0 +1,58 @@
+---
+type: docs
+title: "JetStream KV"
+linkTitle: "JetStream KV"
+description: Detailed information on the JetStream KV state store component
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-nats-jetstream-kv/"
+---
+
+## Component format
+
+To setup a JetStream KV state store create a component of type `state.jetstream`. See [this guide]({{< ref "howto-get-save-state.md#step-1-setup-a-state-store" >}}) on how to create and apply a state store configuration.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: state.aws.dynamodb
+  version: v1
+  metadata:
+  - name: natsURL
```

Comment:
Q: Is there a "name" field required here also or not?

Reply:
Most just checking. It may be good to have the compatibility, but if there are no real-world scenarios for this compatibility (someone moving between the components) then I leave this out.